regolith/dates.py

```diff
@@ -131,14 +131,16 @@ def last_day(year, month):
     """
     return calendar.monthrange(year, month_to_int(month))[1]
 
-def get_dates(thing):
+def get_dates(thing, date_field_prefix=None):
     '''
     given a dict like thing, return the items
 
     Parameters
     ----------
     thing: dict
       the dict that contains the dates
+    date_field_prefix: string (optional)
+      the prefix to look for before the date parameter
```

Comment:
please can you give a concrete example, which is the easiest for people to follow